tests/impls/gen_detector_prop_sal/test_drise_scoring.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import pytest
+import os
+
+from xaitk_saliency.impls.gen_detector_prop_sal.drise_scoring import DRISEScoring
+
+from xaitk_saliency import GenerateDetectorProposalSaliency
+from smqtk_core.configuration import configuration_test_helper
+from tests import DATA_DIR, EXPECTED_MASKS_4x6
+
+
+class TestDRISEScoring:
+
+    def test_init_(self) -> None:
+        """
+        Test if implementation is usable.
+        """
+        impl = DRISEScoring()
+        assert impl.is_usable() and isinstance(impl, GenerateDetectorProposalSaliency)
+
+    def test_default_param(self) -> None:
+        """
+        Test default construction.
+        """
+        impl = DRISEScoring()
+        assert impl.proximity_metric == 'cosine'
+
+    def test_get_config(self) -> None:
+        """
+        Test expected configuation behavior.
+        """
+        impl = DRISEScoring('euclidean')
+        for i in configuration_test_helper(impl):
+            assert i.proximity_metric == 'euclidean'
+
+    def test_metric_args(self) -> None:
+        """
+        Test non-default metric type.
+        """
+        impl = DRISEScoring('hamming')
+        assert impl.proximity_metric == 'hamming'
+
+    def test_shape_sanity(self) -> None:
+        """
+        Test basic scoring with a single feature for broadcasting sanity check.
+        """
+        impl = DRISEScoring()
+        np.random.seed(2)
```

Comment:
I took a quick look at the link, but was unclear about how to proceed. Do I just initialize an instance of `DRISEScoring` with the random seed set (e.g. `self.impl = DRISEScoring()` and `np.random.seed(2)`), and then use that internal instance everywhere?

Reply:
Just the `np.random.seed(2)` part because it is more setting up a per-test background state. Different tests use different initialiations of DRISEScoring, so just retaining one common construction is not really applicable here.

Reply:
Can I propose removing the `np.random.seed(2)` call? I think we can take it out without any loss of generality, since setting the seed isn't critical for testing the results (they are essentially shape and dimension checks where the entries of the matrices themselves are not critical).

Reply:
Sure, that's probably OK for now. If there are any hickups in testing/behavior we can revisit this.

Reply:
I ran the test locally, and nothing came up- thanks!